tests/js/browser-binder-tests.js

```diff
@@ -0,0 +1,639 @@
+// Test "binder" components using `gpii-test-browser`.
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+require("gpii-test-browser");
+gpii.tests.browser.loadTestingSupport();
+
+var path    = require("path");
+var url     = require("url");
+var fileUrl = url.resolve("file://", path.resolve(__dirname, "../static/tests-binder.html"));
+
+fluid.registerNamespace("gpii.binder.tests");
+
+// TODO:  If we find ourselves making model changes often, move this to `gpii-test-browser` and polish it up for wider use.
+gpii.binder.tests.applyChange = function (componentPath, modelPath, newValue) {
+    var component = fluid.getGlobalValue(componentPath);
+    component.applier.change(modelPath, newValue);
+};
+
+
+fluid.defaults("gpii.binder.tests.caseHolder", {
```

Comment:
As we did with `gpii-express` and `kettle`, I would imagine putting a "why this instead of that" bit of wording in the docs for `gpii-test-browser` and `gpii-intern` (or whatever we decide to make that uses the WebDriver API), something like:

> You should use this package if you need to test keyboard navigation between components.  For unit tests of individual components, you should use QUnit.